File: Python For Everyone, Horstmann/Chapter2-Programming-with-Numbers-and-Strings/P2.11.py
# Write a program that asks the user to input
# • The number of gallons of gas in the tank
# • The fuel efficiency in miles per gallon
# • The price of gas per gallon
# Then print the cost per 100 miles and how far the car can go with the gas in the tank.

gallons = float(input("Enter number of gallons in tank: "))
fuelEfficiency = float(input("Enter fuel efficiency in miles per gallons: "))
pricePerGal = float(input("Enter price of gas per gallon: "))

costPer100miles = 100 / fuelEfficiency * pricePerGal
print("The cost per 100 miles is $%.2f" % costPer100miles)

milesGas = fuelEfficiency * gallons
print("You can make it %.2f 5miles on that amount of gas." % milesGas)

File: Big-Java-for-Everyone-Late-Objects-Horstmann/Chapter-2-Fundamental-Data-Types/P2_11.java
/*Write a program that asks the user to input
 • The number of gallons of gas in the tank
 • The fuel efficiency in miles per gallon
 • The price of gas per gallon
Then print the cost per 100 miles and how far the car can go with the gas in the tank.*/


import java.util.Scanner;


public class P2_11 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Gas in tank: ");
        double gas_tank = input.nextDouble();
        System.out.print("Fuel efficiency: ");
        double fuel_efficiency = input.nextDouble();
        System.out.print("Price of gas: ");
        double gas_price = input.nextDouble();
        input.close();
        double cost_100_miles = 100 / fuel_efficiency * gas_price;
        System.out.printf("Cost per 100 miles: %.2f\n", cost_100_miles);
        double max_distance = fuel_efficiency * gas_tank;
        System.out.println("Maximum distance: " + max_distance);
    }
}
